### PR TITLE
Added Keploy 

### PR DESCRIPTION
**Pull request by Arindam200**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [ ] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [ ] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [ ] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [ ] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/keploy.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+description: "Keploy is a developer-centric backend testing tool. It makes backend tests with built-in-mocks, faster than unit tests, from user traffic, making it easy to use, powerful, and extensible.."
+licenses:
+  - Apache-2.0
+platforms:
+  - Java
+  - Python
+  - Nodejs
+  - Go
+  - JavaScript
+  - Docker
```
Comment (Rabenherz112) [inline software/keploy.yml:13]:
```suggestion
platforms:
  - Docker
  - Go
```

File: software/keploy.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+tags:
+  - Automation
+  - Testing  
+  - API Management
+depends_3rdparty: false
+demo_url: "https://www.youtube.com/watch?v=23yQaY81Zho"
```
Comment (Rabenherz112) [inline software/keploy.yml:19]:
```suggestion

```

File: software/keploy.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  - Java
+  - Python
+  - Nodejs
+  - Go
+  - JavaScript
+  - Docker
+tags:
+  - Automation
+  - Testing  
+  - API Management
```
Comment (Rabenherz112) [inline software/keploy.yml:17]:
```suggestion
tags:
  - Software Development - Testing
```

File: software/keploy.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+depends_3rdparty: false
```
Comment (Rabenherz112) [inline software/keploy.yml:18]:
```suggestion
```

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]

**Review by nodiscc — COMMENTED** [review]
This seems to be a purely command-line application - we don't list those. Or am I missing something, such as a self-hosted web interface that allows you to view test results?

**Comment by Arindam200** [comment]
> This seems to be a purely command-line application - we don't list those. Or am I missing something, such as a self-hosted web interface that allows you to view test results?

It has a self-hosted interface for the enterprise users.

**Comment by nodiscc** [comment]
> It has a self-hosted interface for the enterprise users.

Where can I find the source code for this self-hosted enterprise edition, and what is the license?

**Review by ryangurn — APPROVED** [review]
The YML looks as expected and the URLs do exist as expected.

**Comment by nodiscc** [comment]
> It has a self-hosted interface for the enterprise users.
>> Where can I find the source code for this self-hosted enterprise edition, and what is the license?

No reply for > a month, closing

**Pull request closed without merging**